index.html

```diff
@@ -7,6 +7,7 @@
 
 <div class="container target">
 	<h1>Welcome to the openHAB 2 Documentation!</h1>
+	<p>The <strong>open H</strong>ome <strong>A</strong>utomation <strong>B</strong>us (openHAB, <em>pronounced ˈəʊpənˈhæb</em>) is an open source, technology agnostic home automation platform which runs as the center of your smarthome!</p>
```

Comment:
http://docs.openhab.org/images/logo.png
I think it would be more in line to write "smart home"